### PR TITLE
Fix default theme compatibility with npm 8

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | We had some problems running npm on default theme with node 16 and npm 8, this should fix dependencies mistakes
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Could probably be tested by a dev, the legacy BO pages should be as usual, default theme should be building properly
| Possible impacts? | Default theme assets

### CODE REVIEW

**Comment by atomiix** [comment]
@NeOMakinG there's a .DS_Store file that has been added

File: admin-dev/themes/default/.eslintrc.js [hunk 1]
```diff
@@ -18,15 +18,42 @@ module.exports = {
     parser: 'babel-eslint',
   },
   extends: [
-    'prestashop',
+    'airbnb-base',
```
Comment (Progi1984) [inline admin-dev/themes/default/.eslintrc.js:21]: Why this change ?

Comment (NeOMakinG) [reply admin-dev/themes/default/.eslintrc.js:21]: This comes from a public discussion with @SharakPL 

It's a pain in the ass to manage dependency upgrades with a common configuration package, for like 8 rules, it's really not worth

Comment (Progi1984) [reply admin-dev/themes/default/.eslintrc.js:21]: :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-functional can it be tested by the developer?

**Comment by NeOMakinG** [comment]
> @PrestaShop/qa-functional can it be tested by the developer?

I guess yes

**Comment by Robin-Fischer-PS** [comment]
Hi @NeOMakinG @kpodemski ,

Sorry for missing your ping : Yes it can be tested by a dev, thanks !

**Comment by atomiix** [comment]
I've been able to compile assets of the default theme with both npm 7 & 8. This PR is QA ✔️

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @NeOMakinG!
